lib/matplotlib/table.py

```diff
@@ -216,13 +269,21 @@ def add_cell(self, row, col, *args, **kwargs):
         """ Add a cell to the table. """
         xy = (0, 0)
 
-        cell = Cell(xy, *args, **kwargs)
+        cell = CustomCell(xy, visible_edges=self._drawLines, *args, **kwargs)
         cell.set_figure(self.figure)
         cell.set_transform(self.get_transform())
 
         cell.set_clip_on(False)
         self._cells[(row, col)] = cell
 
+    @property
+    def drawLines(self):
```

Comment:
Sorry, I wasn't clear.  I was referring only to the name. `drawLines` sounds like a command--so it evokes the name of a function that will draw something.  But what it really is is just a specification of edges to be drawn, so just plain `edges` seems like a clearer label.